lib/monitor.js

```diff
@@ -40,14 +41,28 @@ Monitor.prototype.createMonitor = function (opt) {
 };
 
 /**
+ * Utility function for converting object into array of tags in the
+ * ['key1:value1', 'key2:value2'] format
+ */
+ Monitor.prototype._toTags = function (json) {
+  return Object.keys(json).map(function(key) {
+    return key + ':' + json[key];
+  });
+};
+
```

Comment:
Does it matter if values or keys contain colons? Do they need to be escaped or replaced somehow?